core/src/main/java/dk/alexandra/fresco/framework/builder/numeric/DefaultPreprocessedValues.java

```diff
@@ -0,0 +1,60 @@
+package dk.alexandra.fresco.framework.builder.numeric;
+
+import dk.alexandra.fresco.framework.DRes;
+import dk.alexandra.fresco.framework.value.SInt;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Default implementation which should only be used if for some reason preprocessing is not
+ * possible.
+ * 
+ * @author Kasper Damgaard
```

Comment:
Please do not use the author tag. We are not using it consistently, and potentially, many authors will be involved in a file. If you want to see who wrote code there is git for that.